Src/StartMenu/StartMenuDLL/MenuContainer.cpp

```diff
@@ -7401,35 +7401,46 @@ bool CMenuContainer::HasMoreResults( void )
 
 RECT CMenuContainer::CalculateWorkArea( const RECT &taskbarRect )
 {
-	RECT rc=s_MenuLimits;
-	if ((s_TaskBarEdge==ABE_LEFT || s_TaskBarEdge==ABE_RIGHT) && GetSettingBool(L"ShowNextToTaskbar"))
+	RECT rc;
+	if (GetSettingBool(L"AlignToWorkArea"))
```

Comment:
Personally, I'd use `if (!GetSettingBool(L"AlignToWorkArea"))`.
So that the diff will be smaller and easier to review.
Also _if_ branch usually represents the typical case.

Though I'm ok with actual change as well.

Reply:
(could i am also do, but the Programmer/Writer should it do.. 
i have write and "[_]viewed" rights at the Files .., just so by side info :stuck_out_tongue_winking_eye:  )